docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md

```diff
@@ -0,0 +1,85 @@
+# get_raw_transaction
+The  `get_raw_transaction`  method takes coin also confirmed tx_hash as input, and returns its full signed raw transaction hex.
```

Comment:
Transaction should not be confirmed. It might be in mempool/pending.
@smk762 Is `method takes coin also confirmed tx_hash` also good from the language perspective?

Reply:
How's this?

The  `get_raw_transaction`  method takes `coin` and `tx_hash` as input, and returns the full signed raw transaction hex for any transaction that is confirmed or within the mempool.

Reply:
updated